src/VisualStudio/Core/SolutionExplorerShim/BrowseObjectAttributes.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.LanguageServices.SolutionExplorer;
+using System;
+using System.ComponentModel;
+using System.Globalization;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.SolutionExplorer
+{
+    /// <summary>
+    /// The attribute used for adding localized display names to properties
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    internal sealed class BrowseObjectDisplayNameAttribute : DisplayNameAttribute
```

Comment:
Since you are just introducing this attribute, there is as yet no code that looks for this attribute, evaluates its DisplayName property, and displays it in the UI, right?